build-system/babel-plugins/babel-plugin-transform-jss/test/fixtures/transform-assertions/should-transform-jss-var/output.mjs

```diff
@@ -1,4 +1,4 @@
-var _classes = JSON.parse("{\"button\":\"button-21aa4a8\",\"CSS\":\".button-21aa4a8 {\\n  font-size: 12px;\\n}\"}");
+var _classes = JSON.parse("{\"button\":\"button-21aa4a8\",\"CSS\":\".button-21aa4a8{font-size:12px}\\n\"}");
```

Comment:
I'm surprised the trailing `\\n` is still here.

Reply:
🤷

Reply:
And we add a new test case that has multiple classes? I wan to see if whitespace between rules is stripped.

Reply:
Can modify an existing test case with a second ruleset

Reply:
Sure